readme.md

```diff
@@ -37,24 +37,20 @@ Type: `Object`
 ##### timeout
 
 Type: `number`<br>
-Default: `2000`
+Default: `5000`
 
-Milliseconds to wait for a server to respond. This option is only supported in Node.js.
+Milliseconds to wait for a server to respond.
 
-##### hostnames
+##### version
```

Comment:
Why do we need a version option? I would prefer as little options as possible.

Reply:
Man, if only you guys made the decision for me! :)

Would you be interested in a PR that would make the default option to try ipv6 if ipv4 didn't succeed? The developer/user would have the option to specify the version to not do this fallback